src/appkit.cs

```diff
@@ -23562,14 +23566,14 @@ interface NSFilePromiseProvider : NSPasteboardWriting
 	interface NSFilePromiseProviderDelegate
```

Comment:
This is a github diff but apparently (look at the line numbers even if you expand the diff).

The real code is:

	[Mac (10,12)]
	[Protocol]
	interface NSFilePromiseProviderDelegate
	{